public/manageclaims.php

```diff
@@ -0,0 +1,269 @@
+}
+</script>
+<div id='mainpage'>
+    <div id='fullcontainer'>
+        <?php
+        echo "<h3>Manage Claims</h3>";
+        echo GetGameAndTooltipDiv($gameID, $gameTitle, $gameIcon, $consoleName, false, 64);
+
+        echo "<div class='embedded mb-1'>";
+        echo "<div><b>Field Values:</b></div>";
+        echo "</br><u>Claim Type</u></br>";
+        echo "<b>" . ClaimType::toString(ClaimType::Primary) . "</b> - Developer has the main claim on the game, this takes up a reservation spot.</br>";
+        echo "<b>" . ClaimType::toString(ClaimType::Collaboration) . "</b> - Developer is collaborating with another developer, this does not take up a reservation spot.</br>";
+        echo "</br><u>Claim Type</u></br>";
+        echo "<b>" . ClaimSetType::toString(ClaimSetType::NewSet) . "</b> - Claim is for a game with no core achievements.</br>";
+        echo "<b>" . ClaimSetType::toString(ClaimSetType::Revision) . "</b> - Claim is for a game with core acheivements.</br>";
+        echo "</br><u>Claim Status</u></br>";
+        echo "<b>" . ClaimStatus::toString(ClaimStatus::Active) . "</b> - Claim is currently active.</br>";
+        echo "<b>" . ClaimStatus::toString(ClaimStatus::Complete) . "</b> - Claim has been marked as complete by the developer.</br>";
+        echo "<b>" . ClaimStatus::toString(ClaimStatus::Dropped) . "</b> - Claim has been dropped by the developer.</br>";
+        echo "</br><u>Special</u></br>";
+        echo "<b>" . ClaimSpecial::toString(ClaimSpecial::None) . "</b> - Standard claim taking up a reservation spot.</br>";
+        echo "<b>" . ClaimSpecial::toString(ClaimSpecial::OwnRevision) . "</b> - Own revision claim, does not take up a claim spot.</br>";
+        echo "<b>" . ClaimSpecial::toString(ClaimSpecial::FreeRollout) . "</b> - Free rollout claim, does not take up a claim spot.</br>";
+        echo "<b>" . ClaimSpecial::toString(ClaimSpecial::ScheduledRelease) . "</b> - Set approved for future release, does not take up a claim spot.</br>";
+        echo "</br><u>Claim Date</u></br>";
+        echo "Date the developer made the claim.</br>";
+        echo "</br><u>Expiration / Completion / Drop Date</u></br>";
+        echo "Date the claim will expire, has been completed or was dropped depending on the claim status.</br>";
+        echo "</div></br>";
+
+        echo "<div class='table-wrapper'><table><tbody>";
+        echo "<th colspan='2'>" . ClaimSorting::toString(ClaimSorting::UserDescending) . "</th>";
+        echo "<th>" . ClaimSorting::toString(ClaimSorting::ClaimTypeDescending) . "</th>";
+        echo "<th>" . ClaimSorting::toString(ClaimSorting::SetTypeDescending) . "</th>";
+        echo "<th>" . ClaimSorting::toString(ClaimSorting::ClaimStatusDescending) . "</th>";
+        echo "<th>" . ClaimSorting::toString(ClaimSorting::SpecialDescending) . "</th>";
+        echo "<th>" . ClaimSorting::toString(ClaimSorting::ClaimDateDescending) . " &#9660;</th>";
+        echo "<th>Finished Date</th>";
```

Comment:
Would "End Date" make more sense here? In most cases, it will be Expiration Date except when viewing historical data.

Also, I realize this is an admin page, but do we really want claim date to be editable? Or any field on a non-Active request?

Reply:
"End Date" sounds good to me.

My intent here was so that if an edit needed to be made it could be done by any admin rather than one of the admins with DB access. I don't expect there to be many admin edits outside of handling expired claims and setting the special flag when necessary.